jekyll/_cci2/server-3-overview.adoc

```diff
@@ -238,8 +238,9 @@ but can also be configured to be exported to other places (i.e. Datadog or Prome
 
 | kotsadm-kots
 | Licensing
-| The main KOTS application. Runs the KOTS Admin Console where upgrades and configuration of server take place	No admin console available.
+| The main KOTS application. Runs the KOTS Admin Console where upgrades and configuration of server take place.
 | No upgrades or configuration possible for server.
+No admin console available.
```

Comment:
```suggestion
| No upgrades or configuration possible for server. No admin console available.
```

Reply:
I think this should be in the "what happens if it fails" column?

Reply:
Doesn't that shift it into the Notes column? This is what my version looks like for me in Docker:
![kotsadm](https://user-images.githubusercontent.com/129828/150345258-7a0ce180-e8d0-407a-99cb-407538a21324.png)

Reply:
oh ok cool, the pic looks right so let's go with that, sorry!

Reply:
merge away!

Reply:
Those tables are really eye-boggling! 👀